KIPs/kip-113.md

```diff
@@ -83,6 +83,45 @@ After reading from IKIP113, users must perform the following validations:
 - [KeyValidate](https://www.ietf.org/archive/id/draft-irtf-cfrg-bls-signature-05.html#section-2.5)
 - [PopVerify](https://www.ietf.org/archive/id/draft-irtf-cfrg-bls-signature-05.html#section-3.3.3)
 
+### JSON-RPC API
+
+The following JSON-RPC method for the Klaytn node should be added to provide the registered bls public keys.
+
+- Name: `klay_getBlsInfos`
+- Description: Returns all registered bls public keys and proof-of-possessions of the validators.
+- Parameters:
+    - `number` - (optional) integer or hexadecimal block number, or the string "pending" or "latest".
+- Returns: A map of the addresses and their associated bls public keys, proof-of-possessions and verify error.
```

Comment:
```suggestion
- Returns: A map of the addresses and their associated BLS public keys, proof-of-possessions and verify error.
```

Reply:
Applied in https://github.com/klaytn/kips/pull/159/commits/3364e36fd1e3ffdd671530229053a8928459f08f